autoinstall_snippets/autoinstall_start

```diff
@@ -1,31 +1,26 @@
 #set system_name = $getVar('system_name','')
 #set profile_name = $getVar('profile_name','')
+#if $system_name != ''
+    #set object_type = 'system'
+    #set object_name = $system_name
+#else if $profile_name != ''
+    #set object_type = 'profile'
+    #set object_name = $profile_name
+#else
+    #set object_type = ''
+    #set object_name = ''
+#end if
 #set breed = $getVar('breed','')
 #set srv = $getVar('http_server','')
 #set run_install_triggers = $str($getVar('run_install_triggers',''))
 #set runpre = ""
-#if $system_name != ''
-    ## RUN PRE TRIGGER
-    #if $run_install_triggers in [ "1", "true", "yes", "y" ]
-        #if $breed == 'redhat'
-            #set runpre = "\ncurl \"http://%s/cblr/svc/op/trig/mode/pre/%s/%s\" -o /dev/null" % (srv, "system", system_name)
-        #else if $breed == 'suse'
-            #set runpre = "\ncurl \"http://%s/cblr/svc/op/trig/mode/pre/%s/%s\" -o /dev/null" % (srv, "system", system_name)
-        #else if $breed == 'vmware'
-            #set runpre = "\nwget \"http://%s/cblr/svc/op/trig/mode/pre/%s/%s\" -O /dev/null" % (srv, "system", system_name)
-        #else if $breed == 'debian' or $breed == 'ubuntu'
-            #set runpre = "\nwget \"http://%s/cblr/svc/op/trig/mode/pre/%s/%s\" -O /dev/null" % (srv, "system", system_name)
-        #end if
-    #end if
-#else if $profile_name != ''
+#if $object_type != ''
     ## RUN PRE TRIGGER
-    #if $run_install_triggers in [ "1", "true", "yes", "y" ]
+    #if $run_install_triggers == "True"
         #if $breed == 'redhat'
-            #set runpre = "\ncurl \"http://%s/cblr/svc/op/trig/mode/pre/%s/%s\" -o /dev/null" % (srv, "profile", profile_name)
-        #else if $breed == 'suse'
```

Comment:
Now it seems that `-O` is the default for `suse`. Just checked with SLE12SP5 and it's not handling `-O`, just `-o`. Could it be the problem?

Reply:
Ignore the previous comment, please.